### PR TITLE
Unomi supports Swagger

### PR DESCRIPTION
**Pull request by weburnit**
I'm having trouble with features.xml!
unomi.rest must have start=true

### CODE REVIEW

File: kar/src/main/feature/feature.xml [hunk 1]
```diff
@@ -69,7 +71,7 @@
         <bundle start-level="77" start="false">mvn:org.apache.unomi/cxs-geonames-rest/${project.version}</bundle>
         <bundle start-level="77" start="false">mvn:org.apache.unomi/cxs-privacy-extension-services/${project.version}</bundle>
         <bundle start-level="77" start="false">mvn:org.apache.unomi/cxs-privacy-extension-rest/${project.version}</bundle>
-        <bundle start-level="78" start="false">mvn:org.apache.unomi/unomi-rest/${project.version}</bundle>
+        <bundle start-level="78" start="true">mvn:org.apache.unomi/unomi-rest/${project.version}</bundle>
```
Comment (weburnit) [inline kar/src/main/feature/feature.xml]: Somehow unomi-rest can not start along with other bundle! therefore I force it over here!

**Comment by sergehuber** [comment]
Hello, thank you for your contribution. I'll have to review it in more details, but I looked at it quickly and have a question. 

I noticed you re-introduce the CXF logging. This was removed because of issues with JDK 11. Which JDK did you test this with ?

Regards,
  Serge...

**Comment by sergehuber** [comment]
Hello, I was also working on OpenAPI support on my side. 

Here are the changes I did:

[UNOMI-____Swagger_&_OpenAPI_integration.patch.txt](https://github.com/apache/unomi/files/4386672/UNOMI-____Swagger_._OpenAPI_integration.patch.txt)

Maybe we could somehow merge these two approaches?

File: kar/src/main/feature/feature.xml (hunk 1 above)
Comment (sergehuber) [reply kar/src/main/feature/feature.xml]: Are you using unomi:start command after Karaf startup ? If you are I thing you will not need to force the start in the feature file. 

The bundles are not started by default to allow for migrations to be possible.

Comment (sergehuber) [reply kar/src/main/feature/feature.xml]: But it is possible to auto-start the bundles with a command line parameter however. 
-Dunomi.autoStart=true

**Comment by fpapon** [comment]
@sergehuber I would prefer to add the swagger feature in your patch, we could be have the both features. However I don't know if it make sense to have the both features, may be only OpenAPI is enough.

**Comment by weburnit** [comment]
> Hello, I was also working on OpenAPI support on my side.
> 
> Here are the changes I did:
> 
> [UNOMI-____Swagger_&_OpenAPI_integration.patch.txt](https://github.com/apache/unomi/files/4386672/UNOMI-____Swagger_._OpenAPI_integration.patch.txt)
> 
> Maybe we could somehow merge these two approaches?

That would be great to merge at your end
[swagger.txt](https://github.com/apache/unomi/files/4393582/swagger.txt)

**Comment by sergehuber** [comment]
I've just create a new PR#171 that includes full Swagger/OpenAPI support so I'm closing this one.
https://github.com/apache/unomi/pull/171

**Pull request closed without merging**